static/app/utils/issueTypeConfig/index.tsx

```diff
@@ -35,7 +35,7 @@ type GetConfigForIssueTypeParams = {eventOccurrenceType: number} | IssueCategory
 const BASE_CONFIG: IssueTypeConfig = {
   actions: {
     archiveUntilOccurrence: {enabled: true},
-    delete: {enabled: false},
+    delete: {enabled: true},
```

Comment:
This is the default value which various issue types can override.

Leaving this as `false` makes the test fail.